Automattic/wp-calypso
# Added Tracks events to the JetpackModuleToggle

#### Changes proposed in this Pull Request

This adds Tracks events to the `JetpackModuleToggle` in order to enable tracking of Jetpack module activation/deactivation in Calypso.

#### Testing instructions

1. Switch to the branch and run `npm run start`.
2. Log in to a WordPress.com account that has a Jetpack site with at least a Premium plan
3. Visit `marketing/traffic/:site`
4. Ensure that analytics debugging is activated by running this in the browser console: `localStorage.setItem('debug', 'calypso:analytics:*');`
5. Toggle Jetpack modules on the Traffic page, watching the console output to verify the events. Verify that the `userid`, `blogid`, `path`, and `module` properties are all correct.

The following settings on the Traffic page are Jetpack modules:
1. Ads
2. SEO
3. Site stats
4. Google analytics
5. wp.me shortlinks
6. Generate XML sitemaps
7. Site verification services
